### DISCUSSION TITLE
Token Question

### DISCUSSION THREAD

**Discussion by YogeLiu** [SWHL/TrOCR-Formula-Rec#7] [Q&A]
你好，这个项目比较有意思并且值得初学者学习。 想了解下对于预训练模型的tokenizer是怎么处理的啊， 训练数据中的latex语法是不是需要重新训练token呢？

**Comment by SWHL** [comment]
现在项目中tokenizer部分没有做任何特殊处理，用的就是`trocr`自带的tokenzier。
可以重新训练的，效果肯定会有提升，这一点可以参考[latex-ocr](https://github.com/lukas-blecher/LaTeX-OCR)项目，里面用的BPE算法，就是基于现有数据训练的。
